ni_measurementlink_generator/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "ni_measurementlink_generator"
-version = "1.0.1"
+version = "1.0.2"
```

Comment:
This change is unnecessary. The version will get bumped automatically when we make the next release.